contrib/freshmaker-cli

```diff
@@ -225,6 +248,49 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url, date_from=date.today() - timedelta(7),
+                               date_to=date.today()):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns
+    the list of advisories that are IMPORTANT or CRITICAL and the issue date is between specified
+    dates.
+    """
+    if (date_to - date_from).days > 7:
+        print("The interval between dates should not exceed 7 days")
+        sys.exit(1)
+
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get(errata_url, auth=krb_auth, timeout=30)
+    r.raise_for_status()
+    data = r.json()
+    return [
```

Comment:
Optional: This is a very long list comprehension and makes it hard to read. Consider turning this into a normal for loop for readability.

Reply:
Done